### PR TITLE
Update Configs for Freeze

### PR DESCRIPTION
**Pull request by terencechain**
This PR updates the configs to the following [commit](https://github.com/ethereum/eth2.0-specs/commit/c79f93fe9962bba5ff2d947eea63ab8619f31390)

Updated:
- Added `MaxValidatorsPerCommittee`
- Added `MinGenesisActiveValidatorCount`
- Added `MinGenesisTime`
- Added `JustificationBitsLength`
- Removed `MaxDepositAmount` for `MaxEffectiveBalance`
- Removed unused configs: `RandBytes`, `BatchBlockLimit`, `SyncEpochLimit `, `MaxNumLog2Validators `, `HashCacheSize `

### CODE REVIEW

File: shared/params/config.go [hunk 1]
```diff
@@ -10,29 +10,32 @@ import (
...
+	ChurnLimitQuotient        uint64 `yaml:"CHURN_LIMIT_QUOTIENT"`         // ChurnLimitQuotient is used to determine the limit of how many validators can rotate per epoch.
+	BaseRewardsPerEpoch       uint64 `yaml:"BASE_REWARDS_PER_EPOCH"`       // BaseRewardsPerEpoch is used to calculate the per epoch rewards.
+	ShuffleRoundCount         uint64 `yaml:"SHUFFLE_ROUND_COUNT"`          // ShuffleRoundCount is used for retrieving the permuted index.
+	SecondsPerDay             uint64 `yaml:"SECONDS_PER_DAY"`              // SecondsPerDay defines how many seconds are in a given day.
+	MaxValidatorsPerCommittee uint64 `yaml:"MAX_VALIDATORS_PER_COMMITTEE"` // MaxValidatorsPerCommittee defines the upper bound of the size of a committee.
+	MinGenesisActiveValidatorCount uint64 `yaml:"MIN_GENESIS_ACTIVE_VALIDATOR_COUNT"` // MinGenesisActiveValidatorCount defines how many validator deposits needed to kick off beacon chain.
+	MinGenesisTime            uint64 `yaml:"MIN_GENESIS_TIME"`             // MinGenesisTime defines the lower bound of the genesis time.
```
Comment (prestonvanloon) [inline shared/params/config.go]: Please add more to the comment here. The comment current restates the variable name in a different way. How could the genesis time have a minimum value?

File: shared/params/config.go [hunk 2]
```diff
@@ -10,29 +10,32 @@ import (
...
+	TargetCommitteeSize       uint64 `yaml:"TARGET_COMMITTEE_SIZE"`        // TargetCommitteeSize is the number of validators in a committee when the chain is healthy.
+	MaxIndicesPerAttestation  uint64 `yaml:"MAX_INDICES_PER_ATTESTATION"`  // MaxIndicesPerAttestation is used to determine how many validators participate in an attestation.
+	MinPerEpochChurnLimit     uint64 `yaml:"MIN_PER_EPOCH_CHURN_LIMIT"`    // MinPerEpochChurnLimit is the minimum amount of churn allotted for validator rotations.
+	ChurnLimitQuotient        uint64 `yaml:"CHURN_LIMIT_QUOTIENT"`         // ChurnLimitQuotient is used to determine the limit of how many validators can rotate per epoch.
+	BaseRewardsPerEpoch       uint64 `yaml:"BASE_REWARDS_PER_EPOCH"`       // BaseRewardsPerEpoch is used to calculate the per epoch rewards.
+	ShuffleRoundCount         uint64 `yaml:"SHUFFLE_ROUND_COUNT"`          // ShuffleRoundCount is used for retrieving the permuted index.
+	SecondsPerDay             uint64 `yaml:"SECONDS_PER_DAY"`              // SecondsPerDay defines how many seconds are in a given day.
```
Comment (prestonvanloon) [inline shared/params/config.go]: This is a constant. Can we consider moving these out of the struct? Why do we need it in beacon config anyway?

This is `time.Day / time.Second` and shouldn't be configurable, in my opinion.

Comment (prestonvanloon) [reply shared/params/config.go]: I mean Duration. Not 100%  on the API without looking at it, but this doesn't seem like something that should be configurable unless the earth changes its rotation significantly.

Comment (terencechain) [reply shared/params/config.go]: Agreed, it's a constant and not a config. Do you want to see a `constant.go`? or have a constant struct. Here's more constants that are being defined as a config:

Screen Shot 2019-06-30 at 4 53 21 PM

Comment (terencechain) [reply shared/params/config.go]: I'm going to remove `SecondsPerDay` for now, it's not needed for core logic, we can define this later

**Pull request merged**